platforms/intel-iot/edison/sparkfun_pinmap.go

```diff
@@ -0,0 +1,116 @@
+package edison
+
+var sparkfunPinMap = map[string]sysfsPin{
+  "12": sysfsPin{
```

Comment:
I get what you are saying, seems like a good thing. 

At some point I think I will implement something to strip off unneeded but nice looking alpha prefixes, just to match board labeling, but that is entirely separate.